.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -0,0 +1,52 @@
+---
+name: User Story
+about: Propose an enhancement via a user story
+title: ''
+labels: kind/user-story
+assignees: ''
+
+---
+## User Story
+
+*Using the form below, relate the desired product behavior.  Feel free to
+additional, clarifying details below it if needed.  For more details on user
+stories, see this [article](https://www.atlassian.com/agile/project-management/user-stories).*
+
+**As a** <persona name>
+**I want** <desired behavior>
+**So that** <value of behavior> 
+
+### Test Scenarios
+
+*If desired, add one or more test scenarios required for user story acceptance.
+Ideally, they are executed by automated tests. For more details on Gerkin syntax,
+see this [article](https://cucumber.io/docs/gherkin/reference/)*
+
+**Given** <this setup/context>
+**When** <an action/event occurs>
+**Then** <the expected behavior/outcome>
+
+### Definition of Done
+
+*The following checklist must be completed before this issue can be completed
+and closed.*
+
+- [ ] Changes have sufficient unit and integration test coverage.
```

Comment:
Eventually we'll want to align DoD with what Maciek is putting together. Typically teams create their own Definition of Ready and Definition of Done but I'm happy to work with Maciek's straw man.

Reply:
I just removed this section for now.